modules/host/contractmanager/consts.go

```diff
@@ -60,6 +60,10 @@ const (
 	// which is a high granluarity relative the to the TiBs of storage that
 	// hosts are expected to provide.
 	storageFolderGranularity = 64
+
+	// maxAddSectorBatchThreads is the maximum number of threads updating
+	// sector counters on disk in AddSectorBatch.
+	maxAddSectorBatchThreads = 100
```

Comment:
we typically sort consts alphabetically, prefer if you fix in another pr. I'll go ahead and merge this since it's a trivial fix, and I don't want to cause too much review friction.